src/Documents.ts

```diff
@@ -829,6 +746,128 @@ class Documents implements Pointer.Library, Pointer.Validator, ObjectSchema.Reso
 		return persistedDocument.getFragment( fragment.id );
 	}
 
+	private setDataRequest( uri:string, requestOptions:HTTP.Request.Options, asContainer:boolean ):string {
+		if( RDF.URI.Util.isRelative( uri ) && ! this.context ) throw new Errors.IllegalArgumentError( "This Documents instance doesn't support relative URIs." );
+
+		if ( this.context ) {
+			if ( this.context.auth.isAuthenticated() ) this.context.auth.addAuthentication( requestOptions );
+			uri = this.context.resolve( uri );
+		}
+
+		HTTP.Request.Util.setContentTypeHeader( "application/ld+json", requestOptions );
```

Comment:
Not all the requests need a Content-Type, and sending one in requests without a body could tell the server to try to parse it. Let's remove this line and include it in the methods that need to configure it.